pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -206,6 +203,48 @@ func getMasterMachines(kubeClient client.Client) (*machineapi.MachineList, error
 	return machineList, nil
 }
 
+func createSubnetGatewayGuardSvc(kubeClient client.Client, requestedIP string) (clusterIP net.IP, err error) {
```

Comment:
perhaps it can be renamed to ensureSubnetGatewayGuardSvc as it's finding and creating if not Found?

Reply:
Makes sense, done.